v2/config/manager/manager_auth_proxy_patch.yaml

```diff
@@ -22,5 +22,6 @@ spec:
       - name: manager
         args:
         - "--metrics-addr=127.0.0.1:8080"
+        - "--health-addr=:8081"
```

Comment:
Does this addr need to be exposed in the service too? I assume no but just checking

Reply:
Nope! it gets reached through the container port